syntax = "proto2";
package comm_network;

message Machine {
	required int64 id = 1;
	required string addr = 2;
	optional int32 ctrl_port_agent = 3 [default = -1];
}

message EnvProto {
  repeated Machine machine = 1;
  required int32 ctrl_port = 2;
}
